fiboa_cli/datasets/hr.py

```diff
@@ -141,12 +141,47 @@
         },
         'jpaid': {
             'type': 'string'
+        },
+        'crop_code': {
+            'type': 'int32'
+        },
+        'crop_name': {
+            'type': 'string'
+        },
+        'crop_name_en': {
+            'type': 'string'
         }
     }
 }
 
 
-# Conversion function, usually no changes required
+CATEGORIES = """\
+200;Oranica;Arable land
+210;Staklenici na oranici;Greenhouses on the arable land
+310;Livada;Meadow
+320;Pašnjak;Pasture
+321;Krški pašnjak;Karst pasture
+410;Vinograd;Vineyard
+411;Iskrčeni vinograd;Cleared vineyard
+421;Maslinik;Olive grove
+422;Voćnjak	Poljoprivredno;Orchard
+430;Kulture kratke ophodnje;Short tour cultures
+450;Rasadnik;Nursery
+490;Mješoviti višegodišnji nasadi;Mixed perennial plantations
+900;Ostale vrste uporabe zemljišta;Other types of land use
+910;Privremeno neodržavana parcela;Temporarily unmaintained plot"""
```

Comment:
I'm wondering whether we should store these in separate files...

Reply:
We could store it as a separate file. I made a proposal

Reply:
Thanks